packages/sanes-chrome-extension/src/extension/background/actions/createAccount/index.ts

```diff
@@ -0,0 +1,11 @@
+import { CreateAccountResponse } from '../../messages';
+import { getCreatedPersona } from '../createPersona';
+
+export async function createAccount(): Promise<CreateAccountResponse> {
```

Comment:
I am going to propose, for reducing complexity migrate all communication between the extension -> bacgkround script using the same approach I implemented for getting the requests' queue. As you have already identified, `getBackgroundPage`. 

In this way, we save some back and forth of messages. But we can do it easily in a separate PR. Not urgent now.

Reply:
I agree, let's hope this works nicely. This also removes the limitation of JSON compatible data